### PR TITLE
s6c3a docs updates

### PR DESCRIPTION
**Pull request by dtbuchholz**
## Summary
Some small tweaks to the site:
- Add dependabot to the repo (note: are monthly updates the right frequency?).
- Made some tweaks to the site's metadata (note: not an SEO guru by any means).
- Content adjusted wrt touching multiple tables in single tx, fix for a misformatted markdown link, add Nova to limits, and some verbiage changes.
- Update mermaid diagram font to Poppins (used to be inter but not rendering after default font change to Poppins).
- Minor style changes:
  - Make the active section (dark mode) in the right side's table of contents (via scrolling) more prominent by changing the color of inactive sections to a different color / very light green (matches the light mode's general design).
  - Heavy bold the active page in the left sidebar (previously, semi-bolded) to also make it more prominent. (Alternatively, could do something like the table of contents above where non-active sidebar pages are very light green.)
  - Dark mode code snippet highlighting—it was too dark before and blended it w/ the bg.
  - Landing page's top margin reduced to make it more vertically centered, and the landing image size/top margin also altered on small screens (was too small / close to the "Get started" btn).

### CODE REVIEW

**Comment by dtbuchholz** [comment]
Note: closes https://github.com/tablelandnetwork/docs/issues/72

File: docs/playbooks/sql/README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Tableland doesn't have full SQLite parity. Let's review a TL;DR of what to look
 - Foreign keys constraints are not supported.
 - Some common SQL languages have type support for numbers with decimals (floats / reals), dates / times, and booleans, but Tableland **does not support any of these** (see the [data type docs](/playbooks/sql#data-types)).
 - Subqueries ("nesting" `SELECT` statements within a query) are fully supported in read queries but limited to flattened subqueries for inserts and cannot be used in updates nor deletes.
-- Mutating queries (inserts, updates, deletes) can only touch _one table at a time_ (i.e., can't write to two different tables in the same SQL statement).
+- Mutating queries (inserts, updates, deletes) can touch one or more tables at a time (i.e., write to two different tables in the same transaction).
```
Comment (sanderpick) [inline docs/playbooks/sql/README.md:52]: same here

**Review by sanderpick — COMMENTED** [review]
Looks great... left a couple comments.

File: docs/playbooks/sql/write.md [hunk 2]
```diff
@@ -20,15 +20,15 @@ For these examples, let's assume we own a `my_table` with a schema of `id int, v
 
 ## Inserting data
 
-When you write data, it **can only correspond to a single table** in a single transaction. For example, the following query would fail since it touches two different tables (if submitted in one transaction):
+When you write data, it can touch one or more tables in a single transaction. For example, the following query is totally valid and will insert values into two different tables—assuming there aren't issues like invalid syntax that causes the inserts to fail:
```
[the remark this answers is not in the document]
Comment (dtbuchholz) [reply docs/playbooks/sql/write.md]: Ah, misunderstood that wasn't live yet—moved to https://github.com/tablelandnetwork/docs/pull/74

**Review by sanderpick — APPROVED** [review]
lgtm!

**Comment by dtbuchholz** [comment]
Closes https://github.com/tablelandnetwork/docs/issues/76

**Pull request merged**